core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -216,10 +216,26 @@ object Battle {
 
         // Must come before normal conquest logic so units that cannot capture cities can still destroy them
         // Melee units can capture capitals; any unit with CanDestroyCities can destroy non-capital cities
-        if (defender.isDefeated() && defender.city.canBeDestroyed()) {
+       if (defender.isDefeated() && defender.city.canBeDestroyed()) {
```

Comment:
Unnecessary eyesore - indent no longer a multiple of 4.
```suggestion
        if (defender.isDefeated() && defender.city.canBeDestroyed()) {
```